sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -153,9 +153,7 @@ case class HashAggregateExec(
 
   override def supportCodegen: Boolean = {
     // ImperativeAggregate and filter predicate are not supported right now
```

Comment:
nit: Let's also update the comment here as well.

Reply:
ok, thanks.